buildstream/sandbox/_sandboxbwrap.py

```diff
@@ -58,11 +66,25 @@ def __init__(self, *args, **kwargs):
         host_os, _, _, _, host_arch = os.uname()
         config = self._get_config()
 
-        # We can't do builds for another host or architecture except 32 bit on 64 bit
+        # We can't do builds for another host OS
         if config.build_os != host_os:
             raise SandboxError("Configured and host OS don't match.")
 
         if config.build_arch != host_arch:
+            try:
+                archtest = utils.get_host_tool('arch-test')
```

Comment:
My only concern with this is availability of arch-test. I'm sort of wondering what would be the problem of just removing all this sanitation and only keeping linux32 parts. Build will fail anyway if CPU arches don't match. Why does BuildStream need to check? It's quite complex to do so.

Reply:
Maybe this is fine. We can in our build images replace this with shim that claims everything works